ci-operator/step-registry/cucushift/chainupgrade/toimage/cucushift-chainupgrade-toimage-commands.sh

```diff
@@ -998,6 +1053,13 @@ for target in "${TARGET_RELEASES[@]}"; do
     fi
     check_history
     health_check
+    currentVersion=$(oc version -o yaml | grep openshiftVersion | grep -o '[0-9]*[.][0-9]*' | head -1)
+    currentPlugin=$(oc get network.config.openshift.io cluster -o jsonpath='{.status.networkType}')
+    if [[ ${currentVersion} == "4.16" && ${currentPlugin} == "OpenShiftSDN" ]]; then
+	echo "the cluster is 4.16 and openshift-sdn need to migration to ovn and then upgrade if need"
+	sdn2ovn
```

Comment:
Does this mean after cluster upgrade completed, need to migrate sdn to ovn ?

Reply:
Yes,  since sdn cannot upgrade to 4.17,  so it needs to migration to ovn and then upgrade to 4.17